docs/pages/docs/handbook/package-installation.mdx

```diff
@@ -0,0 +1,180 @@
+import Callout from "../../../components/Callout";
+import HeartIcon from "@heroicons/react/solid/HeartIcon";
+import { Tabs, Tab } from "../../../components/Tabs";
+
+# Package Installation
+
+A package manager (like `npm`) handles two things for you: installing packages and [managing workspaces](/docs/handbook/workspaces).
+
+Turborepo is compatible with three package managers:
+
+- [npm](https://docs.npmjs.com/cli/v8/using-npm/workspaces#description)
+- [Yarn 1](https://classic.yarnpkg.com/lang/en/docs/workspaces/)
+- [pnpm](https://pnpm.io/workspaces)
```

Comment:
we should combine this in a partial somewhere, there's another reference here: https://github.com/vercel/turborepo/blob/651abfced95220a3d87c6c2a50446171e347ef99/docs/pages/docs/getting-started/create-new.mdx?plain=1#L47-L51. I think we kinda sorta do support Yarn2 (berry), but it would be good to have a canonical reference